test/DI.Tests/ServiceCollectionServiceExtensionsTest.cs

```diff
@@ -211,11 +211,14 @@ public static TheoryData TryAddImplementationTypeData
             ServiceLifetime expectedLifetime)
         {
             // Arrange
-            var collection = new ServiceCollection();
-            collection.Add(ServiceDescriptor.Transient(expectedServiceType, expectedServiceType));
+            var collection = new ServiceCollection
+            {
+                ServiceDescriptor.Transient(expectedServiceType, expectedServiceType)
+            };
 
             // Act
             addAction(collection);
+            collection.TryAdd(ServiceDescriptor.Describe(expectedServiceType, expectedImplementationType, expectedLifetime));
```

Comment:
I don't like adding this like if accomplishes nothing but disabling a warning, which is false positive in this case.